include/osgText/Text

```diff
@@ -73,7 +73,7 @@ public:
      * placing text against noisy backgrounds.
      * The color of the background shadow text is specified by setBackdropColor().
      * DROP_SHADOW_BOTTOM_RIGHT will draw backdrop text to the right and down of
-     * the normal text. Other DROW_SHADOW_* modes do the same for their repective directions.
+     * the normal text. Other DROW_SHADOW_* modes do the same for their respective directions.
```

Comment:
DROW ?

Reply:
It was written that way before, but nevertheless I will correct it, thanks.